pkg/deviceshifu/utils/testdata/pythoncustomizedhandlersfortest/customized_handlers.py

```diff
@@ -0,0 +1,31 @@
+import json
```

Comment:
Make new directories here `mv customized_handlers.py ./example/humidity/humidity_handlers.py`

Reply:
I think we should keep it as it is now, because this handlers.py is for testing purpose and should not have anything to do with example files.